### PR TITLE
main terraform and docker code changes item-handler

### PR DESCRIPTION
**Pull request by drobledomunoz-ch**
JU-768 
need to be confirmed the load balancer and health check path

### CODE REVIEW

**Comment by drobledomunoz-ch** [comment]
treated as consuming service, no load balancer required.

**Comment by drobledomunoz-ch** [comment]
Successfully passed the terraform-runners in local

image

**Comment by drobledomunoz-ch** [comment]
The terraform-runner is fine with no lb configuration as needed 

image

File: README.md [hunk 1]
```diff
@@ -27,3 +27,23 @@ Path | Method | Description
...
 [3]: https://git-scm.com/downloads
+
+## Terraform ECS
+### What does this code do?
+The code present in this repository is used to define and deploy a dockerised container in AWS ECS.
+This is done by calling a [module](https://github.com/companieshouse/terraform-modules/tree/main/aws/ecs) from terraform-modules. Application specific attributes are injected and the service is then deployed using Terraform via the CICD platform 'Concourse'.
+Application specific attributes | Value                                | Description
+:---------|:-----------------------------------------------------------------------------|:-----------
+**ECS Cluster**        |order-service                                      | ECS cluster (stack) the service belongs to
+**Load balancer**      |{env}-apichgovuk <br> {env}-apichgovuk-private                                              | The load balancer that sits in front of the service
```
Comment (chillier-ch) [inline README.md:38]: Just need to remove this

Comment (drobledomunoz-ch) [reply README.md]: Well seen, makes totally sense. Thank you.

**Pull request merged**